lib/nerves_hub_user_api/org_user.ex

```diff
@@ -0,0 +1,78 @@
+defmodule NervesHubUserAPI.OrgUser do
+  @moduledoc """
+  Manage OrgUsers on NervesHub
+
+  Path: /orgs/:org_name/users
+  """
+
+  alias NervesHubUserAPI.{Auth, API, Org}
+
+  @path "users"
+  @roles [:admin, :delete, :write, :read]
+
+  @doc """
+  List all users for an org.
+
+  Verb: GET
+  Path: /orgs/:org_name/users
+  """
+  @spec list(atom() | binary(), NervesHubUserAPI.Auth.t()) :: {:error, any()} | {:ok, any()}
+  def list(org_name, %Auth{} = auth) do
+    API.request(:get, path(org_name), "", auth)
+  end
+
+  @doc """
+  Add a user to the org with a role.
+
+  Verb: POST
+  Path: /orgs/:org_name/users
+  """
+  @spec add(atom() | binary(), atom() | binary(), atom(), NervesHubUserAPI.Auth.t()) ::
```

Comment:
Orgs and products are all `String.t()` types. I don't think `atoms()` are appropriate but it doesn't look like they're needed.